packages/@react-spectrum/s2/src/CardView.tsx

```diff
@@ -262,7 +263,7 @@ export const CardView = /*#__PURE__*/ (forwardRef as forwardRefType)(function Ca
   // ActionBar cannot be inside the GridList due to ARIA and focus management requirements.
   if (props.renderActionBar) {
     return (
-      <div ref={domRef} className={style({position: 'relative', overflow: 'clip', size: 'fit'})}>
+      <div ref={domRef} className={UNSAFE_className + mergeStyles(style({position: 'relative', overflow: 'clip', size: 'fit'}), styles)} style={UNSAFE_style}>
```

Comment:
I think you still want to do a safe merge of styles with allowed overrides, so you may want to pull out this `style` like the other overridable ones `style(..., getAllowedOverrides({height: true}))`